addons/account/models/chart_template.py

```diff
@@ -482,6 +482,14 @@ def try_loading(self, company=False, install_demo=True):
         if not company:
             if request and hasattr(request, 'allowed_company_ids'):
                 company = self.env['res.company'].browse(request.allowed_company_ids[0])
+            elif self.country_id:
+                if self.country_id in (
+                    self.env.company.country_id,
+                    self.env.company.account_fiscal_country_id,
+                ):
+                    company = self.env.company
+                else:
+                    return
```

Comment:
We also want to install the CoA if there is no country set on the company, for instance on the main company without demo data
```suggestion
                company = self.env.company
                company_countries = company.country_id + company.account_fiscal_country_id
                if company_countries and self.country_id not in company_countries:
                    return
```